src/controllers/update/overseas.entity.update.details.controller.ts

```diff
@@ -37,9 +40,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       ...entity,
       ...principalAddress,
       ...serviceAddress,
-      pageParams: {
```

Comment:
Need to confirm that isRegistration is not used now. Both entity.hmtls

Reply:
Used by different controllers. The other html file - `includes/entity.html`- has a isRegistration check but the flag is being passed in by the `overseas.entity.review.controller`. @corybudd-ch worked in this area for UAR-206